docs/css/labels.css

```diff
@@ -0,0 +1,51 @@
+.issue.label-needs-triage .x2{
```

Comment:
would be nice if we can talk about css selector sepcificity

Reply:
yes, I fully agree. Feel free to suggest things and commit changes.